debian/clearwater-snmpd.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
         cp /etc/snmp/snmpd.conf.clearwater-snmpd /etc/snmp/snmpd.conf
 
         # Create the folder to store the statistics UNIX domain sockets.
+        rm -rf /var/run/clearwater
```

Comment:
Why did the C++ code not have permission to delete a socket - didn't it create the socket in the first place. Are you sure there wasn't something else going on (e.g. the C++ code was accidentally run as a root one time and created a socket that socket that a correctly started C++ process couldn't delete)?